packages/frontend-2/components/automate/functions/page/Header.vue

```diff
@@ -126,5 +128,12 @@ if (import.meta.client) {
     },
     { immediate: true }
   )
+  watch(
+    () => route.query['automateBetaRedirect'] as Nullable<string>,
+    (isRedirect) => {
+      if (!isRedirect?.length) return
+      mixpanel.track('Automate Beta Visit Redirected')
```

Comment:
you should always wipe these tracking query params after u process them otherwise you risk processing them multiple times